src/maconn.ts

```diff
@@ -28,7 +57,11 @@ export class WebRTCMultiaddrConnection implements MultiaddrConnection {
   }
 
   async close (err?: Error | undefined): Promise<void> {
-    log.error('error closing connection', err)
+    if (err !== undefined) {
+      log.error('error closing connection', err)
+    }
```

Comment:
might be nice to have a `non-err` `log.debug` equivalent for sanity checking while debugging?